app/src/main/java/io/github/joshkergan/giftr/people/PeopleAdapter.java

```diff
@@ -71,8 +71,9 @@ public long getItemId(int position) {
     @Override
     public int getItemCount() {
         if (validData && c != null){
-            return c.getCount();
-        }else{
+            int count = c.getCount();
+            return count;
```

Comment:
my bad I think it was the debugger skipping to incorrect lines and me assuming that changing this fixed it.